python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py

```diff
@@ -221,6 +222,8 @@ def __init__(self,
                 be different. In address this problem, fuse the pattern before
                 quantization. Default False.
             is_use_cache_file(bool, optional): This param is deprecated.
+            quant_blocks(int|list, optional): The bolck id list with quantization.
```

Comment:
In static graph,  it's hard for users to know the block id. why not use 'name_scope of skip_quant' ?

Reply:
delete it.